neuralprophet/TorchProphet.py

```diff
@@ -0,0 +1,553 @@
+from neuralprophet.forecaster import NeuralProphet
+import logging
+
+log = logging.getLogger("NP.forecaster")
+
+
+class TorchProphet(NeuralProphet):
+    """
+    Prophet wrapper for the NeuralProphet forecaster.
+
+    Parameters
+    ----------
+    growth: String 'linear', 'logistic' or 'flat' to specify a linear, logistic or
+        flat trend.
+    changepoints: List of dates at which to include potential changepoints. If
+        not specified, potential changepoints are selected automatically.
+    n_changepoints: Number of potential changepoints to include. Not used
+        if input `changepoints` is supplied. If `changepoints` is not supplied,
+        then n_changepoints potential changepoints are selected uniformly from
+        the first `changepoint_range` proportion of the history.
+    changepoint_range: Proportion of history in which trend changepoints will
+        be estimated. Defaults to 0.8 for the first 80%. Not used if
+        `changepoints` is specified.
+    yearly_seasonality: Fit yearly seasonality.
+        Can be 'auto', True, False, or a number of Fourier terms to generate.
+    weekly_seasonality: Fit weekly seasonality.
+        Can be 'auto', True, False, or a number of Fourier terms to generate.
+    daily_seasonality: Fit daily seasonality.
+        Can be 'auto', True, False, or a number of Fourier terms to generate.
+    holidays: pd.DataFrame with columns holiday (string) and ds (date type)
+        and optionally columns lower_window and upper_window which specify a
+        range of days around the date to be included as holidays.
+        lower_window=-2 will include 2 days prior to the date as holidays. Also
+        optionally can have a column prior_scale specifying the prior scale for
+        that holiday.
+    seasonality_mode: 'additive' (default) or 'multiplicative'.
+    seasonality_prior_scale: Not supported for regularisation in NeuralProphet,
+        please use the `_reg` args instead.
+    holidays_prior_scale: Not supported for regularisation in NeuralProphet,
+        please use the `_reg` args instead.
+    changepoint_prior_scale: Not supported for regularisation in NeuralProphet,
+        please use the `_reg` args instead.
+    mcmc_samples: Not required for NeuralProphet
+    interval_width: Float, width of the uncertainty intervals provided
```

Comment:
update description for NP: mention these are symmetrically converted to quantiles and point towards that interface being an alternative.